src/utils/getLang.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+module.exports = (filePathStem, collection) => {
+    return filePathStem.replace(new RegExp(`\/collections\/${collection}\/([A-Za-z-]*)\/[a-z0-9-]*`, "g"), "$1");
```

Comment:
Is the `/collections/` path segment an 11ty convention or requirement?

Reply:
It's a choice I made for Trivet. I didn't have a `collections` directory for weavly.org and the `src` directory is much more cluttered as a result: https://github.com/codelearncreate/weavly.org/tree/main/src

Reply:
Hmm.. I was going to suggest that you parameterize the base path; however, in skimming through the code again, it would seem that someone would have to have to rewrite quite a bit to use a different directory name/structure. I suppose this makes `collections` a requirement for using trivet. As long as that's clear in the docs, I guess this is fine.